### ISSUE TITLE
Pinned tabs off screen in 119

### ISSUE THREAD

**Issue by xR2** [Aris-t2/CustomCSSforFx#621]
1. Does the issue occur without any files inside /chrome/ folder?
No

2. Issue description:
The pinned tabs disappeared, only accessible via Ctrl+Tab. This issue happens  only after I updated firefox to version 119. It seems to be the same problem as reported last year [here](https://github.com/Aris-t2/CustomCSSforFx/issues/495).

The only settings I have is on `userChrome.css` , line **585**  `@import "./css/tabs/tabs_below_navigation_toolbar.css"; /**/`

3. Does the issue occur with the most recent version of CustomCSSforFx (release/test/dev build)?
Yes. I tested custom_css_for_fx_v4.4.1.zip and custom_css_for_fx_v4.4.1v2.zip

4. Screenshots showing the issue (drag & drop images into this post):
![artwork14](https://github.com/Aris-t2/CustomCSSforFx/assets/17341234/9aec7edb-d33f-40ef-937a-050aa1df946f)

5. System information
Windows 10 Pro 22H2 (build 19045.3448)
Scaling: 225%
Firefox version: 119.0b3 (64-bit)
Firefox theme : build in dark
Is 'about:config > toolkit.legacyUserProfileCustomizations.stylesheets' set to 'true': **Yes**.

**Comment by Aris-t2** [comment]
Unfortunately I can not test your configuration. Does this also happen, if your scaling is set to 100%?

Simulating bigger scaling via `about:config > layout.css.devPixelsPerPx > 2.25` does not offer such an issue for me:
![image](https://github.com/Aris-t2/CustomCSSforFx/assets/6013569/5bd8ccb2-39a6-4cae-9808-3413f4faf91e)

**Comment by xR2** [comment]
Maybe I understood this wrongly. My scale at 225% is setted at OS level.
At `layout.css.devPixelsPerPx` I have it set as **-1**. Changing to 2 or 1 doesn't fix it.

If I switch to ` @import "./css/tabs/tabs_below_titlebar_above_navigation_toolbar.css"; /**/` the issue doesn't happen.

**Comment by Aris-t2** [comment]
I meant "does this happen too, if you set your OS scaling to 100 %"?. 
The devpix settings can only simulate scaling, but is does note have to be identical to what the OS does.

This issue does not happen here. Nothing changed codewise, that would point to such an issue.

**Comment by xR2** [comment]
> I meant "does this happen too, if you set your OS scaling to 100 %"?

Unfortunately it still happens.

**Comment by Aris-t2** [comment]
Sorry, I don't know why this happens on your end.
The only suggest left is setting up chrome folder with latest version of this projects files again.

**Comment by xR2** [comment]
I tried deleting the chrome folder, restarting browser, then adding the latest version and restarting again: nothing.
Then I decided to play with `TAB appearance` and it seemed to be like the issue only happens with the _Classic tabs appearance known from Firefox 4-28 _ , however, after going to "no appearance" setting, and back with classic plus the `/tabs_below_navigation_toolbar.css` it is now working.

Maybe I should have started without any settings after the update and then edit the css afterwards.
